appinventor/docs/html/reference/components/sensors.html

```diff
@@ -137,6 +137,7 @@ <h1 id="sensors">Sensors</h1>
   <li><a href="#Hygrometer">Hygrometer</a></li>
   <li><a href="#LightSensor">LightSensor</a></li>
   <li><a href="#LocationSensor">LocationSensor</a></li>
+  <li><a href="#MagneticFieldSensor">MagneticFieldSensor</a></li>
```

Comment:
The anchor ID here is MagneticFieldSensor, but the id of the documentation section is MagneticField. They need to be the same for the anchor link to work.

Reply:
@SusanRatiLane it should be now solved. Installed ruby and bundler so I can run the docs scripts. I have also committed a little fix to the Serial component documentation which was not generated using the script.